src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -429,6 +430,19 @@ define(function (require, exports, module) {
         callback();
     }
 
+    // Based on http://stackoverflow.com/questions/21797299/convert-base64-string-to-arraybuffer
```

Comment:
True, it does seem a bit odd. I did some digging into the code and the quick explanation is that there are other functions in `FilerFileSystem.js` that we use that aren't in `FilerUtils.js`. So if we put it in `FilerUtils.js`, then we'll need two requires or getModule statements that we're accomplishing with one instead right now.

Extra:
We actually use [writeFile](https://github.com/timmoy/brackets/blob/7c744eb5675c7243d749c62a26c4cf07d9c80b41/src/filesystem/impls/filer/FilerFileSystem.js#L257) in both of the save functions for the [camera](https://github.com/timmoy/brackets/blob/7c744eb5675c7243d749c62a26c4cf07d9c80b41/src/extensions/default/BrambleUrlCodeHints/camera/index.js#L49) and [image viewer](https://github.com/timmoy/brackets/blob/7c744eb5675c7243d749c62a26c4cf07d9c80b41/src/editor/Image.js#L26). Since we need both the base64ToBuffer function and writeFile function for saving, it reduces the overhead of pulling files by including the function we ported in another file that has code we need to work with. It is also quite relevant to the FileSystem, so it's placement still makes sense unlike if it was placed into a non file system file.